### PR TITLE
Misc. UI adjustments [Dev-30.0] #80

### PR DESCRIPTION
**Pull request by grollk**

### CODE REVIEW

File: src/components/SideMenu.tsx [hunk 1]
```diff
@@ -135,15 +135,8 @@ const SideMenu: React.FC = () => {
...
-            <IonHeader>
-                <IonToolbar>
-                    <IonTitle>
-                        Menu
-                    </IonTitle>
-                </IonToolbar>
-            </IonHeader>
             <IonContent>
-                <IonList>
+                <IonList class="side-menu">
```
Comment (hstaatsxx) [inline src/components/SideMenu.tsx:139]: In Figma, the side menu has a slightly brighter color than the background.
Do we need to adjust this part as well? In its current state, it is not implemented.

**Review by hstaatsxx — CHANGES_REQUESTED** [review]
The dark mode issue teaser looks good!
I commented some issues that I found :) please take a look! thx.

File: src/pages/ArticleTeaser.tsx [hunk 2]
```diff
@@ -143,7 +143,7 @@ export const ArticleTeaser: React.FC<Props> = ({ issue, article }) => {
           </div>
           <div className="mr-4 text-sm text-right h-4 text-white z-40 -mt-6">{valFor(language, 'article.photo')}: {article.image_credit}</div>
           <div className="relative h-full w-auto" >
-            <div className="_image_container h-full m-0 p-0" onClick={() => openBrowser(article.link)} style={style}>
+            <div className="_image_container h-full m-0 p-0 image-gradient" onClick={() => openBrowser(article.link)} style={style}>
```
[the remark this answers is not in the document]
Comment (grollk) [reply src/pages/ArticleTeaser.tsx:146]: This Bug is not caused by these changes. It is also reproducible in the main branch. I would suggest fixing it in a separate issue.

File: src/components/SideMenu.tsx (hunk 1 above)
Comment (grollk) [reply src/components/SideMenu.tsx:139]: The background of the side menu mockup in figma is not taken from the app, I would suggest sticking with the color difference that is implemented currently.

**Review by hstaatsxx — APPROVED** [review]
The bug seems to be from SwiperJS with auto-height.
https://github.com/nolimits4web/swiper/issues/4599
PR is approved. We need to open another bugfix ticket.

**Pull request merged**